rmgpy/qm/molecule.py

```diff
@@ -409,18 +409,27 @@ def parse(self):
         parser.logger.setLevel(
             logging.ERROR
         )  # cf. http://cclib.sourceforge.net/wiki/index.php/Using_cclib#Additional_information
```

Comment:
Updated link to cclib documentation
```suggestion
        )  # cf. https://cclib.github.io/index.html#how-to-use-cclib
```